sql/src/main/java/org/apache/druid/sql/calcite/rel/Windowing.java

```diff
@@ -232,10 +202,48 @@ public static Windowing fromCalciteStuff(
         throw new ISE("No processors from Window[%s], why was this code called?", window);
       }
 
-      ops.add(new WindowOperatorFactory(
+      wrapperObjs.add(new WindowGroupProcessorWrapper(windowGroup, new WindowOperatorFactory(
```

Comment:
why not pass `group` instead of `windowGroup` ?

Reply:
changed to pass group, this would allow us to not compute it again while creating sort and partition operators.